index.js

```diff
@@ -2,3 +2,5 @@
 const runApplescript = require('run-applescript');
 
 module.exports = bundleName => runApplescript(`get id of application "${bundleName}"`);
```

Comment:
Can you extract the script here into a variable and reuse it in the sync method?